docs/recipes/storage/use-sqlprovider-ssdt.md

```diff
@@ -0,0 +1,199 @@
+# Using SQLProvider SQL Server SSDT
+
+## Creating a "SafeTodo" Database with Azure Data Studio
+
+### Connecting to a SQL Server Instance
+1) In the "Connections" tab, click the "New Connection" button
+
+![image](https://user-images.githubusercontent.com/1030435/111041187-26777d00-8405-11eb-9260-5d885b0ff640.png)
```

Comment:
The images should probably be committed to the repo instead of just linking to githubusercontent

Reply:
The images were added using the cool new GitHub feature that lets you paste them directly into the editor. 

According to this SO post, these files do not expire and cannot be removed by the customer:
https://stackoverflow.com/questions/46374918/do-images-in-user-images-githubusercontent-com-have-expire-time